Rego/DefenderConfig.rego

```diff
@@ -273,33 +275,48 @@ tests[{
 # MS.DEFENDER.2.1v1
 #--
 
-# TODO: update this policy to match emerald baseline
-# The following check is from pre-emerald 2.5 first bullet,
-# which is similar but needs some adjustments.
+InpersonationProtectionErrorMsg(StrictInpersonationProtection, StandardInpersonationProtection) := Description if {
+    Description := "No users are included for targeted user protection in Strict policy."
+    StrictInpersonationProtection.Result == false
+    StandardInpersonationProtection.Result == true
+}
 
-ProtectedUsersPolicies[{
-    "Name" : Policy.Name,
-	"Users" : Policy.TargetedUsersToProtect,
-    "Action" : Policy.TargetedUserProtectionAction
-}] {
-    Policy := input.anti_phish_policies[_]
-    Policy.Enabled # filter out the disabled policies
-    Policy.EnableTargetedUserProtection # filter out the policies that have impersonation protections disabled
-    count(Policy.TargetedUsersToProtect) > 0 # filter out the policies that don't list any protected users
+InpersonationProtectionErrorMsg(StrictInpersonationProtection, StandardInpersonationProtection) := Description if {
```

Comment:
"Inpersonation" is actually spelled "Impersonation"

Reply:
Fix pushed, please confirm